.circleci/ds005_outputs.txt

```diff
@@ -17,9 +17,7 @@ smriprep/sub-01/anat/sub-01_desc-preproc_T1w.json
 smriprep/sub-01/anat/sub-01_desc-preproc_T1w.nii.gz
 smriprep/sub-01/anat/sub-01_dseg.nii.gz
 smriprep/sub-01/anat/sub-01_from-fsnative_to-T1w_mode-image_xfm.txt
-smriprep/sub-01/anat/sub-01_from-MNI152NLin2009cAsym_to-T1w_mode-image_xfm.h5
```

Comment:
Currently, there are no `--output-spaces` defined at CircleCI. Since the normalization is tested anyways with ds054, I would keep this to exercise this edge case.